examples/stream_download.rb

```diff
@@ -0,0 +1,23 @@
+dir = File.expand_path(File.join(File.dirname(__FILE__), '..', 'lib'))
+require File.join(dir, 'httparty')
+require 'pp'
+
+class StreamDownload
+  include HTTParty
+  stream_body = true
```

Comment:
I don't believe this is correct. [stream_body is an option for requests](https://github.com/jnunemaker/httparty/blob/0d808c7f3a68b8d091506e22003fc00dd84bf6a6/spec/httparty_spec.rb#L777-L788), not a class level method. Also, class level methods don't use ='s as that would mean assignment, not a method call.